src/app/core/http-services/item-navigation.service.ts

```diff
@@ -199,4 +199,45 @@ export class ItemNavigationService {
     return (isSkill(type)) ? this.getRootSkills() : this.getRootActivities();
   }
 
+  getNavigationNeighbors(itemRoute: ItemRoute): Observable<{ parent: ItemRoute|null, left: ItemRoute|null, right: ItemRoute|null }> {
+    const parentId = itemRoute.path[itemRoute.path.length - 1];
+
+    // Root activity => no parent/left/right activity
+    if (!parentId) return of({ parent: null, left: null, right: null });
+
+    return this.getNavDataFromChildRoute(parentId, itemRoute).pipe(map(navParent => {
+      const index = navParent.items.findIndex(item => item.id === itemRoute.id);
+      if (index === -1) throw new Error(`Unexpected: item is missing from its parent children list`);
```

Comment:
Unexpected backticks when single quotes may be used.

Reply:
fixed